ui/libs/dotcms-js/src/lib/core/util/response-view.ts

```diff
@@ -13,11 +13,14 @@ import { DotCMSResponse } from '../core-web.service';
  * }
  * </code>
  */
+
+// TODO: Refactor this to an interceptor
 export class ResponseView<T = any> {
     private bodyJsonObject: DotCMSResponse<T>;
     private headers: HttpHeaders;
 
     public constructor(private resp: HttpResponse<DotCMSResponse<T>>) {
+        console.info('ResponseView class', resp);
```

Comment:
remove.

Reply:
I will add a rule to lint to not allow console.